### PR TITLE
feat(drs): add new drs job resource

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
add new drs job resource

### CODE REVIEW

File: docs/resources/drs_job.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+  provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the job name. The name consists of 4 to 50 characters, starting with
+ a letter. Only letters, digits, underscores (\_) and hyphens (-) are allowed.
+
+* `type` - (Required, String, ForceNew) Specifies the job type. Changing this parameter will create a new
+ resource. The options are as follows:
+  + **migration**: Online Migration.
+  + **sync**: Data Synchronization.
+  + **cloudDataGuard**: Disaster Recovery.
```
Comment (ShiChangkuo) [inline docs/resources/drs_job.md:77]: cloudDataGuard is not supported in FE

File: docs/resources/drs_job.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+* `engine_type` - (Required, String, ForceNew) Specifies the migration engine type.
+ Changing this parameter will create a new resource. The options are as follows:
+  + **mysql**:  MySQL migration, MySQL synchronization use.
+  + **mongodb**: Mongodb migration use.
+  + **cloudDataGuard-mysql**: Disaster recovery use.
+  + **gaussdbv5**: GaussDB (for openGauss) synchronization use.
```
Comment (ShiChangkuo) [inline docs/resources/drs_job.md:84]: gaussdbv5 is not supported in FE

File: docs/resources/drs_job.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+* `tags` - (Optional, Map, ForceNew) Specifies the key/value pairs to associate with the DRS job.
+ Changing this parameter will create a new resource.
+
+* `force_destroy` - (Optional, Bool) Specifies whether to forcibly destroy the job even if it is running.
+ The default value is `false`.
+
+The `db_info` block supports:
+
+* `engine_type` - (Required, String, ForceNew) Specifies the engine type of database. Changing this parameter will
+ create a new resource. The options are as follows: `mysql`, `mongodb`, `gaussdbv5`.
```
Comment (ShiChangkuo) [inline docs/resources/drs_job.md:159]: ditto

File: docs/resources/drs_job.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/drs_job.md]: fixed

File: docs/resources/drs_job.md (hunk 2 above)
Comment (Jason-Zhang9309) [reply docs/resources/drs_job.md]: fixed

File: docs/resources/drs_job.md (hunk 3 above)
Comment (Jason-Zhang9309) [reply docs/resources/drs_job.md]: fixed

**Pull request merged**